CheckDrive.Api/CheckDrive.Application/Services/Authorization/JwtHandler.cs

```diff
@@ -0,0 +1,57 @@
+﻿using CheckDrive.Domain.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+
+namespace CheckDrive.Application.Services.Authorization;
+public sealed class JwtHandler(IOptions<JwtOptions> options,
+    UserManager<IdentityUser>userManager)
+{
+    private readonly JwtOptions _options = options.Value 
+        ?? throw new ArgumentNullException(nameof(options));
+    private readonly UserManager<IdentityUser> _userManager = userManager
+        ?? throw new ArgumentNullException(nameof(userManager));
```

Comment:
this class should be a helper class and registered as a singleton, it shouldn't use UserManager to get roles.